text/0000-project-groups.md

```diff
@@ -0,0 +1,385 @@
+# Project Group RFC
+
+# Summary
+
+- Formalize project groups as groups dedicated to completing specific projects
```

Comment:
This language makes it sound as if these "project groups" are ephemeral. Am I misunderstanding this, or is this indeed the desired use?

Reply:
It depends on the group. Some groups will be ephemeral, as they have a set task, and once that task is completed so will the project group (e.g. Inline ASM). However there can be projects groups that are formed around a task that has continuous work (e.g. Rustup, Website, Triage).

Reply:
We should explicitly list this below